WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Package and Rate Selection/WooShippingServiceView.swift

```diff
@@ -62,7 +62,6 @@ private struct WooShippingServiceCardListView: View {
                     .fixedSize(horizontal: false, vertical: true) // Prevents card text from being truncated
             }
         }
-        .frame(maxHeight: .infinity, alignment: .top)
```

Comment:
This was previously added to ensure each tab's list was aligned to the top of the tab content area. However, it was adding extra space below the lists. This was fixed directly in `TopTabView` as I can't think of a scenario where we wouldn't want the tab content to appear directly below the tabs (at the top of the tab content area).